x-pack/plugins/secops/server/lib/events/elasticsearch_adapter.ts

```diff
@@ -21,15 +21,15 @@ const EventFieldsMap = {
   'host.ip': 'host.ip',
   'event.category': 'suricata.eve.alert.category',
   'event.id': 'suricata.eve.flow_id',
-  'event.module': 'fileset.module',
+  'event.module': 'event.module',
   'event.type': 'event.type',
   'event.severity': 'suricata.eve.alert.severity',
   'suricata.eve.flow_id': 'suricata.eve.flow_id',
   'suricata.eve.proto': 'suricata.eve.proto',
   'suricata.eve.alert.signature': 'suricata.eve.alert.signature',
   'suricata.eve.alert.signature_id': 'suricata.eve.alert.signature_id',
-  'source.ip': 'source_ecs.ip',
-  'source.port': 'source_ecs.port',
+  'source.ip': 'source.ip',
+  'source.port': 'source.port',
```

Comment:
note: It's almost to the point we won't need mappings anymore, but it is AWESOME during this time of trialing things to have it right now.